### PR TITLE
use dnsConfig on kind jobs to reduce ndots

### PR DESCRIPTION
**Pull request by BenTheElder**
this may help with https://github.com/kubernetes/test-infra/issues/9555

if it does we should look at doing this more widely

/area kind
/area jobs

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kind/kind-postsubmits.yaml [hunk 1]
```diff
@@ -10,3 +10,10 @@ postsubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20180924-8e8468033-experimental
```
Comment (MrHohn) [inline config/jobs/kubernetes-sigs/kind/kind-postsubmits.yaml]: Which base image is this using? Would be great to check if the underlying library would respect the ndots option we set :)

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kind/kind-postsubmits.yaml]: https://github.com/kubernetes/test-infra/tree/master/images/kubekins-e2e ->
https://github.com/kubernetes/test-infra/tree/master/images/bootstrap ->
`FROM debian:jessie`

Comment (MrHohn) [reply config/jobs/kubernetes-sigs/kind/kind-postsubmits.yaml]: Great, debian uses glibc, which I think does repsect the configuration in `/etc/resolv.conf`.

**Comment by BenTheElder** [comment]
FYI @munnerz

**Pull request merged**